src/Illuminate/Queue/RedisQueue.php

```diff
@@ -98,7 +98,7 @@ public function bulk($jobs, $data = '', $queue = null)
     {
         $this->getConnection()->pipeline(function () use ($jobs, $data, $queue) {
             $this->getConnection()->transaction(function () use ($jobs, $data, $queue) {
-                foreach ((array) $jobs as $job) {
```

Comment:
Breaking change. The phpdoc is wrong. You can also call this with a single job, not as an array.